civihr_default_theme/scss/components/_button.scss

```diff
@@ -0,0 +1,7 @@
+.chr-button--inline {
```

Comment:
At best this is a variation of the standard `.btn` bootstrap component, so it's definitely not a `.chr_` custom component (notice that you wrote `.chr-`)

But this seems weirdly specific, with also an explicit reference to a dropdown. What is this used for?